core/src/test/scala/github/ParseGithubOpenApiTest.scala

```diff
@@ -0,0 +1,17 @@
+package net.zhenglai
+package github
+
+import io.swagger.v3.parser.OpenAPIV3Parser
+import org.scalatest.FunSuite
+
+
+class ParseGithubOpenApiTest extends  FunSuite{
+
+  test("parse github open api"){
+    val fileUrl = "https://raw.githubusercontent.com/github/rest-api-description/main/descriptions/ghes-3.1/ghes-3.1.json"
```

Comment:
is the test read from remote github server? It's not pretty stable for unit case scenario